pages/common/omz.md

```diff
@@ -0,0 +1,32 @@
+# omz
+
+> Oh My Zsh command-line tool.
+> More information: <https://github.com/ohmyzsh/ohmyzsh>.
+
+- Update Oh My Zsh:
+
+`omz update`
+
+- Print the changes from the latest update of Oh My Zsh:
+
+`omz changelog`
+
+- Restart the current Zsh session and Oh My Zsh:
+
+`omz reload`
+
+- List plugins:
```

Comment:
```suggestion
- List all available plugins:
```

Reply:
Could be not available?

Reply:
Maybe just 

``` suggestion 
- List all plugins:
```

Reply:
> Could be not available?

No, all plugins in are saved locally and if you remove one, it won't show up in this list anymore.

@CleanMachine1 I like using `available`, because with just `all` it's not that clear what exactly you mean. All available plugins? All installed plugins? All active plugins?